rfcs/20210513-pluggable-profiler-for-tensorflow.md

```diff
@@ -0,0 +1,440 @@
+# Modular TensorFlow profiler C API
```

Comment:
@deven-amd
Please have a look at this RFC. I think it would be cleaner to have all the GPU profilers consolidated behind a common interface.

Reply:
Hi @yisitu, I am working closely with Deven in AMD. Please also CC me for further discussions for the TF profiler. Thanks.

Reply:
Sure!